Makefile

```diff
@@ -1,15 +1,22 @@
 # For Linux and MacOS users only and assumes brew is already installed
 TECH_MARKDOWN_FILE := doc/technical-overview.md
-PDF_OUTFILE := dds_web/static/dds-technical-overview.pdf
+TECH_OVERVIEW_PDF := dds_web/static/dds-technical-overview.pdf
+TROUBLE_MARKDOWN_FILE := doc/troubleshooting.md
+TROUBLE_PDF := dds_web/static/dds-troubleshooting.pdf
 
 install_pandoc:
-	brew install pandoc
+	brew install pandoc basictex librsvg
```

Comment:
`basictex` didn't work for me, and I needed to install the full MacTex package. There is a chance that I overlooked something though, so we can leave it like that. Maybe we can add a comment that in case of issues one can install the full package from https://www.tug.org/mactex/mactex-download.html

Reply:
Ah sorry, about the back and forth faff. What is the error message you get? 

I can OFC add the MacTex package instead - my rationale was to avoid downloading such a huge thing.

Reply:
I don't remember the exact error messages, but basically I didn't get `tlmgr` installed with `basictex` only. I guess it is hard to tell without trying on a clean system.

Reply:
Hmm, okay, that would be a weird error because `tlmgr` is installed by default with `basictex`- https://tug.org/mactex/BasicTeX.pdf

Reply:
I actually tested on a fresh MacOS system and the Makefile works without issues as it is.

Reply:
Ah thanks a million. Okay, will clean up commits and merge now